cronjobs/github_api_util.py

```diff
@@ -0,0 +1,44 @@
+# Python 3 module
+# -*- coding: utf-8 -*-
+import json
+import os
+import re
+import requests
+import util
+
+def GetAllIssuesForUBTueRepository(repository):
```

Comment:
Please use type annotations in all of our new Python code.

Reply:
I cannot find annotations anywhere in our code base.

Reply:
That's because we were on Python 2 not that long ago and that did not support type annotations.  In general we want all the support in finding mistakes that our compilers or interpreters provide.